src/Sparkfabrik/Helpers/Traits/Command/Redmine/RedmineSearchTrait.php

```diff
@@ -29,4 +29,57 @@ public function redmineUsersObjectToFirstLastname($users)
         }
         return $ret;
     }
+
+
+    /**
+   * List users.
+   *
+   * @link http://www.redmine.org/projects/redmine/wiki/Rest_Users#GET
+   *
+   * @param object $client Redmine client
+   *
+   * @param array $projectId Redmine projectid used to get users
+   * from membership if current user is not admin
+   *
+   * @param array $params to allow offset/limit (and more) to be passed
+   *
+   * @return array list of users found
+   */
+    public function redmineUsersGetAll($client, $projectId, array $params = array())
```

Comment:
passing the $client object is not really needed, as the trait is ever included within the command class.